test/framework/toy_build.py

```diff
@@ -1536,6 +1536,10 @@ def test_minimal_toolchains(self):
         reprod_ec = os.path.join(self.test_installpath, 'software', 'toy', '0.0', 'easybuild', 'reprod', 'toy-0.0.eb')
         self.assertTrue(os.path.exists(reprod_ec))
 
+        reprod_easyblock = os.path.join(self.test_installpath, 'software', 'toy', '0.0', 'easybuild', 'reprod',
+                                        'easyblocks', 'toy.py')
+        self.assertTrue(os.path.exists(reprod_easyblock))
```

Comment:
@ocaisa Can we come up with an additional test to make sure inheritance is picked up correctly?

Also, make sure `easyblock.py` is *not* being copied, i.e. there's only one file being copied into `reprod/easyblocks` in this case?

Reply:
2nd comment is done, first is a bit more involved.

Reply:
Ok done